packages/lib-classifier/src/components/Classifier/components/QuickTalk/components/PostForm.js

```diff
@@ -49,18 +44,22 @@ function PostForm ({
   let statusText
   if (postCommentStatusMessage) {
     statusText = postCommentStatusMessage
+  } else if (postCommentStatus === asyncStates.initialized) {
+    statusText = t('QuickTalk.status.initialized')
   } else if (postCommentStatus === asyncStates.loading) {
     statusText = t('QuickTalk.status.loading')
+  } else if (postCommentStatus === asyncStates.success) {
+    statusText = t('QuickTalk.status.success')
   }
 
   return (
     <Box
       background={{ dark: 'dark-1', light: 'light-1' }}
       flex={false}
-      pad='small'
+      pad='xsmall'
     >
       {statusText && (
-        <Text>{statusText}</Text>
+        <Text role='status'>{statusText}</Text>
```

Comment:
```suggestion
        <Text role='status' aria-live="polite">{statusText}</Text>
```
[MDN recommends](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#roles_with_implicit_live_region_attributes) explicitly setting politeness for status roles.

Reply:
Having said that, [these screenreader tests](https://www.digitala11y.com/status-role/) say the implicit role is well-supported so maybe that's needed for older screen readers? 🤷‍♂️

Reply:
Oh that's definitely something I missed

The resources I was reading on [W3](https://www.w3.org/TR/wai-aria-1.1/#status) and the [_other_ MDN page](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role) both said that `role=status` elements have `aria-live=polite`

But the [MDN page you shared](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#roles_with_implicit_live_region_attributes) does mention the explicit aria-live=polite is to "To maximize compatibility", so you're probably right that it's needed for older screen readers.

I'll commit the suggestion, thanks Jim 👍